src/spells.h

```diff
@@ -271,13 +424,29 @@ class RuneSpell final : public Action, public Spell
 		uint16_t getRuneItemId() const {
 			return runeId;
 		}
+		void setRuneItemId(uint16_t i) {
+			runeId = i;
+		}
+		bool getHasCharges() const {
```

Comment:
Can't you reuse ``charges``? e.g. ``charges > 0``.

Reply:
done in [5296c38](https://github.com/otland/forgottenserver/pull/2584/commits/5296c38673bb57e34bf5fe3769265a42a8d1dc95) thanks